src/engine/Account.c

```diff
@@ -5484,6 +5484,211 @@ gnc_account_imap_add_account_bayes (GncImportMatchMap *imap,
     LEAVE(" ");
 }
 
+/*******************************************************************************/
+
+static void
+build_bayes_layer_two (const char *key, const GValue *value, gpointer user_data)
+{
+    QofBook     *book;
+    Account     *root;
+    gchar       *kvp_path;
+    gchar       *probability;
+
+    struct kvp_info *kvpInfo_node;
+
+    struct kvp_info *kvpInfo = (struct kvp_info*)user_data;
+
+    // Get the book
+    book = qof_instance_get_book (kvpInfo->source_account);
+    root = gnc_book_get_root_account (book);
+
+    PINFO("build_bayes_layer_two: account '%s', token_count: '%ld'",
+                                  (char*)key, (long)g_value_get_int64(value));
+
+    probability = g_strdup_printf ("%ld", g_value_get_int64 (value));
```

Comment:
A gint64 is a long only in some 64-bit builds; on 32-bit builds it's a long long. GLib provides G_GINT64_FORMAT which always expands to the right value.